testkit/javadsl/src/main/scala/com/lightbend/lagom/internal/testkit/ProducerStubFactory.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.testkit
```

Comment:
About packaging: I focused on the visibility of classes. 
In the scaladsl I could use `private[lagom]` to mark classes in the non-`internal` packages and trust users wouldn't be able to use them. When working in the javadsl I preferred moving anything suposed to be internal to the `internal` package.